ui-admin/src/study/surveys/SurveyEditorView.tsx

```diff
@@ -12,6 +13,25 @@ type SurveyEditorViewProps = {
   onSave: (update: { content: string }) => Promise<void>
 }
 
+type Draft = {
+  content: string
+  date: number
+}
+
+const LoadedDraftModal = ({ onDismiss }: {
+  onDismiss: () => void
+}) => {
+  return <Modal show={true} onHide={onDismiss}>
+    <Modal.Header>
+      <Modal.Title>Survey Draft Loaded</Modal.Title>
+    </Modal.Header>
+    <Modal.Body>
+      <p>Previously unsaved changes to this survey have been automatically loaded.
+        Be sure to save the survey in order to publish your changes.</p>
+    </Modal.Body>
+  </Modal>
+}
```

Comment:
I ran out of time before going OOO tomorrow but I intend to break this out into it's own component and add a red button that allows the user to discard the draft if they don't want it. Right now they're forced to use it unless they're savvy enough to go into local storage to destroy the value